### PR TITLE
Fedora 36 cockpit

### PR DESCRIPTION
**Pull request by jelly**
* [x] image-refresh fedora-36

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-9.
Log: https://logs.cockpit-project.org/logs/image-refresh-2984-20220223-164545/

**Comment by cockpituous** [comment]
image-refresh fedora-36 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-36-20220223-170404

**Comment by cockpituous** [comment]
image-refresh in progress on 1-ci-srv-06.
Log: https://logs.cockpit-project.org/logs/image-refresh-2984-20220224-132222/

**Comment by cockpituous** [comment]
image-refresh fedora-36 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-36-20220224-134141

**Comment by martinpitt** [comment]
Triggering starter-kit and podman, to ensure that there are no regressions.

File: images/scripts/fedora.setup [hunk 1]
```diff
@@ -18,6 +18,7 @@ echo foobar | passwd --stdin root
 # the distribution repository is updated.
 #
 COCKPIT_DEPS="\
+acl \
```
Comment (martinpitt) [inline images/scripts/fedora.setup:21]: This should have gone into `TEST_DEPENDS`, but let's do that in a follow-up (if nothing else has to change)

**Review by martinpitt — APPROVED** [review]
Thanks for the yak shaving! Waiting for regression tests, otherwise LGTM (almost)!

File: images/scripts/fedora.setup (hunk 1 above)
Comment (jelly) [reply images/scripts/fedora.setup:21]: Good catch.. working on a follow up.

**Pull request merged**